test/SlowTests/Tests/TestsInheritanceTests.cs

```diff
@@ -92,7 +92,7 @@ where Filter(method)
 
             var array = types.ToArray();
             const int numberToTolerate = 4712;
-            if (array.Length == numberToTolerate)
+            if (array.Length <= numberToTolerate)
```

Comment:
I don't think this is something we want, better is to adjust `numberToTolerate` in case you moved more tests...
eventually, this `numberToTolerate` will be zero :)

Reply:
done :)